apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -271,14 +271,16 @@ if [[ "$should_use_xcodebuild" == true ]]; then
     exit 1
   fi
 
+  test_product_module_name=$(echo "$test_bundle_name" | /usr/bin/sed -r 's/-/_/g')
+
   readonly xctestrun_file="$test_tmp_dir/tests.xctestrun"
   /usr/bin/sed \
     -e "s@BAZEL_INSERT_LIBRARIES@$xctestrun_libraries@g" \
     -e "s@BAZEL_TEST_BUNDLE_PATH@__TESTROOT__/$test_bundle_name.xctest@g" \
     -e "s@BAZEL_TEST_ENVIRONMENT@$xctestrun_env@g" \
     -e "s@BAZEL_TEST_HOST_BASED@$xctestrun_test_host_based@g" \
     -e "s@BAZEL_TEST_HOST_PATH@$xctestrun_test_host_path@g" \
-    -e "s@BAZEL_TEST_PRODUCT_MODULE_NAME@$test_bundle_name@g" \
+    -e "s@BAZEL_TEST_PRODUCT_MODULE_NAME@$test_product_module_name@g" \
```

Comment:
```suggestion
    -e "s@BAZEL_TEST_PRODUCT_MODULE_NAME@${test_bundle_name//-/_}@g" \
```

the magic of bash

Reply:
You'd think I'd remember this after using bash substitution multiple times in the past week.